CveXplore/common/config.py

```diff
@@ -78,6 +78,8 @@ class Configuration(object):
         "cwe": "https://cwe.mitre.org/data/xml/cwec_latest.xml.zip",
         "capec": "https://capec.mitre.org/data/xml/capec_latest.xml",
         "via4": "https://www.cve-search.org/feeds/via4.json",
+        # See EPSS at https://www.first.org/epss
+        "epss": "https://epss.cyentia.com/epss_scores-current.csv.gz"
```

Comment:
Who is hosting this?

Reply:
https://www.cyentia.com/about/#who-we-are -> It appears to be a cybersecurity research company, but I found the EPSS project on the first.org website, and it mentioned that:

"EPSS is an emerging standard developed by a volunteer group of researchers, practitioners, academics and government personnel. We grant the use of EPSS scores freely to the public, subject to the conditions below. We reserve the right to update the model and these webpages periodically, as necessary, though we will make every attempt to provide sufficient notice to users in the event of material changes. While membership in the EPSS SIG is not required to use or implement EPSS, however, we ask that if you are using EPSS, that you provide appropriate attribution where possible. EPSS can be cited either from this website (e.g. "See EPSS at https://www.first.org/epss), or as: Jay Jacobs, Sasha Romanosky, Benjamin Edwards, Michael Roytman, Idris Adjerid, (2021), Exploit Prediction Scoring System, Digital Threats Research and Practice, 2(3)"

So, if you are concerned about licensing, it should not be a problem.

Reply:
Licensing is not my main concern; I'm more concerned about the site where the epss is downloaded from; isn't there a download available straight from the first.org site?

Reply:
I thought this link was an "external" link, but I just discovered that the link is posted on the first.org site.